pandas/io/tests/parser/multithread.py

```diff
@@ -0,0 +1,97 @@
+# -*- coding: utf-8 -*-
+
+"""
+Tests multithreading behaviour for reading and
+parsing files for each parser defined in parsers.py
+"""
+
+from __future__ import division
+from multiprocessing.pool import ThreadPool
+
+import numpy as np
+import pandas as pd
+import pandas.util.testing as tm
+
+from pandas import DataFrame
+from pandas.compat import BytesIO, range
+
+
+class MultithreadTests(object):
+    def construct_dataframe(self, num_rows):
+
```

Comment:
`construct_dataframe` became a private function outside of `MultithreadTests`.  I also made `generate_multithread_dataframe` a private method of `MultithreadTests` (still needs `self`).